src/Raven.Server/Documents/Queries/OrderByField.cs

```diff
@@ -27,6 +34,8 @@ public OrderByField(QueryFieldName name, OrderByFieldType orderingType, bool asc
 
         public readonly MethodType? Method;
 
+        public readonly string LuceneOrderByName;
```

Comment:
@arekpalinski how this will be in Corax? Can we just call it OrderByName?

Reply:
When using Corax we don't have the notion of `_D`, `_L` fields. For a single numeric field we have string, long and double values. I'm for `OrderByName` or `SearchEngineOrderByName`.

Reply:
Renamed